### PR TITLE
Dependency: timber -> 1.0.0

### PR DESCRIPTION
**Pull request by glennsl**
Depends on https://github.com/revery-ui/revery/pull/781

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,14 +29,14 @@
   "dependencies": {
     "revery": "0.29.0",
     "reason-libvterm": "github:revery-ui/reason-libvterm#e0737be",
-    "isolinear": "^3.0.0"
+    "isolinear": "^3.0.0",
+    "@glennsl/timber": "1.0.0"
   },
   "resolutions": {
-    "revery": "revery-ui/revery#e7bcb42",
+    "revery": "glennsl/revery#a3fd3da",
```
Comment (glennsl) [inline package.json]: TODO: update after merging https://github.com/revery-ui/revery/pull/781

**Comment by glennsl** [comment]
Oh, and of course HEAD here still has breaking changes, so I can't submit a working PR to oni2 for this either.

**Review by bryphe — APPROVED** [review]

**Comment by glennsl** [comment]
We can either wait to merge this until you've updated oni2 with he latest changes here, then I can merge this and update oni2 with he isolinear changes. Or if you want to make those changes as well, you can just merge this and the timber PR whenever.

**Comment by bryphe** [comment]
Thanks @glennsl ! I'll plan on merging this now - i need to upgrade revery for Onivim, and revery already has this upgrade - so all the deps need it too. (I'm upgrading reason-jsonrpc as well here: https://github.com/onivim/oni2/pull/1372)

Once Onivim 2 is upgraded with timber - then we can bring in the isolinear changes across the dependencies (revery-terminal and Onivim 2)

**Pull request merged**